### PR TITLE
Release `v6.0.0-1`

### PR DESCRIPTION
**Pull request by aurexav**

### CODE REVIEW

File: runtime/darwinia/src/lib.rs [hunk 1]
```diff
@@ -83,7 +83,7 @@ pub const VERSION: sp_version::RuntimeVersion = sp_version::RuntimeVersion {
 	spec_name: sp_runtime::create_runtime_str!("Darwinia2"),
 	impl_name: sp_runtime::create_runtime_str!("DarwiniaOfficialRust"),
 	authoring_version: 0,
-	spec_version: 6_0_0_0,
+	spec_version: 6_0_0_1,
```
Comment (boundless-forest) [inline runtime/darwinia/src/lib.rs:86]: We do not need to bump darwinia version? as it has not been launched yet.

Comment (AurevoirXavier) [reply runtime/darwinia/src/lib.rs:86]: Keep the same with the crate version.

Comment (hackfisher) [reply runtime/darwinia/src/lib.rs:86]: How about pangolin?

Comment (AurevoirXavier) [reply runtime/darwinia/src/lib.rs:86]: Pangolin is a testnet, I think I'll upgrade it frequently.

**Pull request merged**